server/openapi_server/test/utils.py

```diff
@@ -18,13 +18,14 @@ def client_note_to_request_dict(note: Note):
 
 
 ANNOTATOR_TYPE_MAP = {
-    'nlpsandbox:physical-address-annotator': (
+    'nlpsandbox:physical-address-nlpsandboxsdk': (
```

Comment:
Not a valid too type.

Reply:
I thought it was weird, too, but it appears to be [what `nlpsandboxclient` expects](https://github.com/nlpsandbox/nlpsandbox-client/blob/d58e400a5c57d8fe4b4ae891fe4b5bc35b9201af/nlpsandboxclient/client.py#L503).

Reply:
Great find guys!  Aggressive search and find issue strikes again.